proposals/lip-0048.md

```diff
@@ -25,15 +25,57 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
+
+### Additional Fees
+
+In principle, any transaction that results in a state increase or significant computational effort could require a fee. Since the current implementation does not provide an automatic way to charge fees in these cases, we introduce the [`payFee`](#payfee) method. 
+
+This method can be invoked by other modules to charge an additional fee when a state increase is triggered during the execution of a transaction, e.g., when a new validator is registered in the PoS module, a new sidechain is registered, or a [user account is initialized][lip-0051#initializeuseraccountinternal] in the Token module: 
+```python
+Fee.payFee(USER_ACCOUNT_INITIALIZATION_FEE)
+```
+In addition, developers may decide to call the `payFee` method for transactions that require extensive computation, e.g., when verifying an inclusion proof, or if any other resource is extensively used during transaction execution.
+
+### Fee Processing
+
+We refer to the total of the minimum fee and the fees processed by the `payFee` method as the "consumed" part of the fee. The remainder of the fee, called the "available" fee, is given to the validator as a reward. In this way, validators are assumed to prioritize transactions with a higher available fee in case of network congestion. [After the transaction is executed](#after-command-execution), consumed fee is either burned or transferred to a dedicated pool address. 
+
+#### Fee Pool
+
+The option to transfer the consumed fees to a dedicated pool address `ADDRESS_FEE_POOL` could be beneficial for sidechains, as fees collected this way can be used as an incentive for validators and other protocol participants for various purposes: Funding a community pool for on-chain governance spending, sharing funds proportionally among validators per round, incentivizing ecosystem developers, supporting chain maintenance and improvement, rewarding users' staking and voting activities, etc.
+
+In the Lisk mainchain, the consumed fees are always burned. This reduces the total supply, creating a deflationary pressure that works against inflation.
+
+### Cross-chain Message Fees
+
+Cross-chain messages, or CCMs, facilitate interoperability by enabling cross-chain transfer of data. They require an associated fee, distinct from the usual transaction fee. Similar to transaction fees, the consumed fee can either be burned or transferred to a fee pool depending on the configuration of the chain. The available fee is rewarded to a CCM relayer to cover the cost of CCM execution and to incentivize participation in CCM relaying.
+
+#### Fee Token ID
```

Comment:
Now that we found the right structure, I think this could be added in the initial presentation of trs fees, for example after the link to LIP 68 (meaning adding this sentence there instead of having a separate sub-sub-section for it)

Reply:
Done.